commit 46c4205e82efcaf1165df8593eaeb2115eef3056
Author: Jan RÄ™korajski <baggins@pld-linux.org>
Date:   Sun Apr 24 10:28:38 2016 +0200

    Revert "python: make installed modules find by default on Debian and Ubuntu"
    
    This reverts commit 1f113f6bc81edabbbda7e14d58b10ac02d7a1137.

diff --git a/m4/python.m4 b/m4/python.m4
index 32c30e2..5c58851 100644
--- a/m4/python.m4
+++ b/m4/python.m4
@@ -216,26 +216,6 @@ AC_DEFUN([AM_PATH_PYTHON],
   # Substituting python_exec_prefix_subst.
   AC_SUBST([PYTHON_EXEC_PREFIX], [$am_python_exec_prefix_subst])
 
-  # Factor out some code duplication into this shell variable.
-  am_python_setup_sysconfig="\
-import sys
-# Prefer sysconfig over distutils.sysconfig, for better compatibility
-# with python 3.x.  See automake bug#10227.
-try:
-    import sysconfig
-except ImportError:
-    can_use_sysconfig = 0
-else:
-    can_use_sysconfig = 1
-# Can't use sysconfig in CPython 2.7, since it's broken in virtualenvs:
-# <https://github.com/pypa/virtualenv/issues/118>
-try:
-    from platform import python_implementation
-    if python_implementation() == 'CPython' and sys.version[[:3]] == '2.7':
-        can_use_sysconfig = 0
-except ImportError:
-    pass"
-
   dnl emacs-page Set up 4 directories:
 
   dnl 1. pythondir: where to install python scripts.  This is the
@@ -251,14 +231,7 @@ except ImportError:
    else
      am_py_prefix=$am_cv_python_prefix
    fi
-   am_cv_python_pythondir=`$PYTHON -c "
-$am_python_setup_sysconfig
-if can_use_sysconfig:
-  sitedir = sysconfig.get_path('purelib', vars={'base':'$am_py_prefix'})
-else:
-  from distutils import sysconfig
-  sitedir = sysconfig.get_python_lib(0, 0, prefix='$am_py_prefix')
-sys.stdout.write(sitedir)"`
+   am_cv_python_pythondir=`$PYTHON -c "import sys; from distutils import sysconfig; sys.stdout.write(sysconfig.get_python_lib(0,0,prefix='$am_py_prefix'))" 2>/dev/null`
    #
    case $am_cv_python_pythondir in
    $am_py_prefix*)
@@ -293,14 +266,7 @@ sys.stdout.write(sitedir)"`
    else
      am_py_exec_prefix=$am_cv_python_exec_prefix
    fi
-   am_cv_python_pyexecdir=`$PYTHON -c "
-$am_python_setup_sysconfig
-if can_use_sysconfig:
-  sitedir = sysconfig.get_path('platlib', vars={'platbase':'$am_py_exec_prefix'})
-else:
-  from distutils import sysconfig
-  sitedir = sysconfig.get_python_lib(1, 0, prefix='$am_py_exec_prefix')
-sys.stdout.write(sitedir)"`
+   am_cv_python_pyexecdir=`$PYTHON -c "import sys; from distutils import sysconfig; sys.stdout.write(sysconfig.get_python_lib(1,0,prefix='$am_py_exec_prefix'))" 2>/dev/null`
    #
    case $am_cv_python_pyexecdir in
    $am_py_exec_prefix*)
